"strings" package is not imported in the generated .pb.gorm.go file

While trying to compile the following proto, the generated .pb.gorm.go file does not import "strings" package even though the package is used in .pb.gorm.go

```
syntax = "proto3";

package service;

import "github.com/infobloxopen/protoc-gen-gorm/options/gorm.proto";
import "github.com/infobloxopen/protoc-gen-gorm/types/types.proto";

service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  option (gorm.opts).ormable = true;
  string name = 1;
  gorm.types.JSONValue tags = 2;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
```

This results in a failure while building the application
```
# github.com/learning/test.service/pkg/pb
pkg/pb/service.pb.gorm.go:155:22: undefined: strings
```

--- a/plugin/handlergen.go
+++ b/plugin/handlergen.go
@@ -284,6 +284,7 @@ func (p *OrmPlugin) generateApplyFieldMask(message *generator.Descriptor) {
 			p.P(`}`)
 		} else if strings.HasSuffix(fieldType, protoTypeJSON) && !field.IsRepeated() {
 			p.P(`if !updated`, ccName, ` && strings.HasPrefix(f, prefix+"`, ccName, `") {`)
+			p.UsingGoImports("strings")
 			p.P(`patchee.`, ccName, ` = patcher.`, ccName)
 			p.P(`updated`, ccName, ` = true`)
 			p.P(`continue`)

